wiki/markdown/index.md

```diff
@@ -0,0 +1,18 @@
+## Introduction
+
+REMReM (**REST Mailbox for Registered Messages**) is a set of tools that can be used to generate validated Eiffel messages and publish them on a RabbitMQ message bus. They can be run as micro services or as stand-alone CLI versions. For more details on the micro services and the REMReM design, see [Eiffel REMReM](https://github.com/eiffel-community/eiffel-remrem)
+
+Eiffel REMReM Generate takes a partial message in JSON format, validates it and adds mandatory fields before outputting a complete, valid Eiffel message.
+
+
+## Pre-requisites
+
+*   JDK 8
+*   Tomcat 8
+
+For supporting latest features, Eiffel REMReM Generate should use the latest version of [Eiffel REMReM Semantics](https://eiffel-community.github.io/eiffel-remrem-semantics/).
```

Comment:
The link should point to the GitHub repository instead, since the version will change often.